src/app/components/ScrollablePromo/Promo/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, string, oneOfType, object } from 'prop-types';
+import styled from '@emotion/styled';
+import pathOr from 'ramda/src/pathOr';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { getSerifBold } from '@bbc/psammead-styles/font-styles';
+import { C_GREY_6, C_WHITE } from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_0_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { ServiceContext } from '#contexts/ServiceContext';
+import filterForBlockType from '#lib/utilities/blockHandlers';
+
+const C_GREY_8 = '#202224';
+
+const Link = styled.a`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => service && getSerifBold(service)}
+  width: 100%;
+  word-break: break-word;
+  text-overflow: ellipsis;
+  text-decoration: none;
+
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
```

Comment:
Are we okay with this requirement only being met on webkit?

Also, are we sure we need to apply the prefix here manually?  Normally, Emotion would take care of this for us